captum/optim/_param/image/transform.py

```diff
@@ -214,6 +215,30 @@ def forward(self, input: torch.Tensor) -> torch.Tensor:
         return self.translate_tensor(input, insets)
 
 
+class ScaleInputRange(nn.Module):
+    """
+    Multiplies the input by a specified multiplier for models with input ranges other
+    than [0,1].
+    """
+
+    def __init__(self, multiplier: float = 1.0) -> None:
+        super().__init__()
+        self.multiplier = multiplier
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        return x * self.multiplier
+
+
+class RGBToBGR(nn.Module):
+    """
+    Converts RGB images to BGR by switching the red and blue channels.
+    """
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        assert x.dim() == 4
+        return x[:, [2, 1, 0]]
```

Comment:
The dimension of channel is the 2nd dimension , right ? We expect bsz x channel x height x width ? 
Perhaps we can document that ?

Reply:
It's a transform for image tensors, so I expect that they'll be in the form of NCHW. I've added an assert to check that the number of channels is 3.